kfdefs/overlays/osc/osc-cl2/superset/kustomization.yaml

```diff
@@ -4,8 +4,8 @@ kind: Kustomization
 namespace: odh-superset
 resources:
   - kfdef.yaml
+  - configmaps
```

Comment:
This isn't really important but why was this change made?

Reply:
it was a failed attempt to re order alphabetically :) 
fixed now